src/test/java/uk/gov/hmcts/sptribs/exception/InvalidResourceExceptionTest.java

```diff
@@ -10,15 +10,16 @@
 
 class InvalidResourceExceptionTest {
     @Test
-    void testInvalidResourceExceptionThrownForNonExistentFile() throws Exception {
+    void testInvalidResourceExceptionThrownForNonExistentFile() throws RuntimeException {
```

Comment:
The unwanted/unused Exception methods can be removed from Exception class InvalidResourceException.